sdk/BUILD.gn

```diff
@@ -1758,6 +1758,7 @@ if (is_ios || is_mac) {
           "objc/components/video_codec/RTCVideoEncoderH265.h",
           "objc/components/video_codec/RTCVideoEncoderH265.mm",
         ]
+        defines += ["RTC_USE_H265"]
```

Comment:
Any reason for iOS you use RTC_USE_H265 while for others you use WEBRTC_USE_H265?

Reply:
iOS uses WEBRTC_USE_H265 as well. But this change only compiled on Linux with and without GN arg rtc_use_h265.